src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -49,124 +59,123 @@ public static async Task Main(string[] args)
-                        .Concat(excludedOpIndices)
-                        .GroupBy(x => x)
-                        .Where(gp => gp.Count() < 2)
-                        .Select(gp => gp.Key)
-                        .ToArray();
-
-                clientOperations = opIndices.Select(i => clientOperations[i]).ToArray();
-                break;
-            default:
-                Console.Error.WriteLine("Cannot specify both -ops and -xops flags simultaneously");
-                return;
+            Console.Error.WriteLine("Must specify a valid run mode");
+            return;
         }
 
-        Console.WriteLine("       .NET Core: " + Path.GetFileName(Path.GetDirectoryName(typeof(object).Assembly.Location)));
-        Console.WriteLine("    ASP.NET Core: " + Path.GetFileName(Path.GetDirectoryName(typeof(WebHost).Assembly.Location)));
-        Console.WriteLine("          Server: " + (httpSys ? "http.sys" : "Kestrel"));
-        Console.WriteLine("      Server URL: " + serverUri);
-        Console.WriteLine("         Tracing: " + (logPath == null ? (object)false : logPath.Length == 0 ? (object)true : logPath));
-        Console.WriteLine("     ASP.NET Log: " + aspnetLog);
-        Console.WriteLine("     Concurrency: " + concurrentRequests);
-        Console.WriteLine("  Content Length: " + maxContentLength);
-        Console.WriteLine("    HTTP Version: " + httpVersion);
-        Console.WriteLine("        Lifetime: " + (connectionLifetime.HasValue ? $"{connectionLifetime}ms" : "(infinite)"));
-        Console.WriteLine("      Operations: " + string.Join(", ", clientOperations.Select(o => o.name)));
-        Console.WriteLine("     Random Seed: " + seed);
-        Console.WriteLine("    Cancellation: " + 100 * cancellationProbability + "%");
-        Console.WriteLine("Query Parameters: " + numParameters);
-        Console.WriteLine();
+        if (config.ServerUri.Scheme != "https")
+        {
+            Console.Error.WriteLine("Server uri must be https.");
+            return;
+        }
 
-        if (listOps)
+        if (config.ListOperations)
         {
             for (int i = 0; i < clientOperations.Length; i++)
             {
-                Console.WriteLine($"{i} = {clientOperations[i].name}");
+                Console.WriteLine(clientOperations[i].name);
             }
             return;
         }
 
-        if ((runMode & RunMode.both) == 0)
+        // derive client operations based on arguments
+        (string name, Func<RequestContext, Task> op)[] usedClientOperations = (config.OpIndices, config.ExcludedOpIndices) switch
         {
-            Console.Error.WriteLine("Must specify a valid run mode");
-            return;
-        }
+            (null, null) => clientOperations,
+            (int[] incl, null) => incl.Select(i => clientOperations[i]).ToArray(),
+            (_, int[] excl) =>
+                Enumerable
+                .Range(0, clientOperations.Length)
+                .Concat(excl)
```

Comment:
Seems like the Except operator would accomplish this more easily

Reply:
Fixed